app/platform_api/schema/invoice.rb

```diff
@@ -5,17 +5,17 @@ def self.id_ref
   nil
 end
 
-  # The GroupKey uniquely identifies a single Lockstep Platform account.  All records for this
-  # account will share the same GroupKey value.  GroupKey values cannot be changed once created.
-  #             
+  # The GroupKey uniquely identifies a single Lockstep Platform account.  All records for this
```

Comment:
If the swagger.json was updated, can you keep the updated version in this commit. It's fine if the other models are updated too.

Reply:
I did not make any changes. These must be just spaces. I dont see any changes in terms of words if I read